### PR TITLE
test: add snapshot for cli flags

### PR DESCRIPTION
**Pull request by snitin315**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [x] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?
No

### Motivation / Use-Case
Update cli flags docs.

### Breaking Changes
None

### Additional Info
No

### CODE REVIEW

**Comment by snitin315** [comment]
Should we add a snapshot test for cli-flags?

**Comment by alexander-akait** [comment]
@snitin315 yep, we should test it

**Review by alexander-akait — DISMISSED** [review]

**Review by anshumanv — COMMENTED** [review]
need update snapshots

File: test/cli/__snapshots__/cli.test.js.snap.webpack5 [hunk 1]
```diff
@@ -74,11 +74,11 @@ exports[`CLI --host localhost: stderr 1`] = `
 `;
 
 exports[`CLI should generate correct cli flags 1`] = `
-"Usage: webpack serve|s [entries...] [options]
+"[1mUsage:[22m webpack serve|s [entries...] [options]
 
 Run the webpack dev server.
 
-Options:
+[1mOptions:[22m
```
Comment (anshumanv) [inline test/cli/__snapshots__/cli.test.js.snap.webpack5:81]: need strip ansi

Comment (snitin315) [reply test/cli/__snapshots__/cli.test.js.snap.webpack5]: Done

**Review by anshumanv — APPROVED** [review]
LGTM, /cc @alexander-akait

File: test/server/utils/__snapshots__/runOpen.test.js.snap.webpack4 [hunk 2]
```diff
@@ -0,0 +1,68 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (alexander-akait) [inline test/server/utils/__snapshots__/runOpen.test.js.snap.webpack4:1]: This tests will be removed in master, need rebase

**Comment by alexander-akait** [comment]
@snitin315 Sorry, again, need to rebase, let's merge your PR to avoid these problems

**Review by alexander-akait — APPROVED** [review]
Let's merge when it will be green

**Review by anshumanv — APPROVED** [review]

**Pull request merged**